saleor/core/management/commands/fix_undiscounted_total_prices.py

```diff
@@ -0,0 +1,157 @@
+from collections import defaultdict
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.models import Exists, F, OuterRef
+
+from ....celeryconf import app
+from ....order.models import Order, OrderLine
+from ....order.tasks import send_order_updated
+
+ORDERS_UPDATE_BATCH_SIZE = 500
```

Comment:
Please do not just magic numbers

Reply:
It's a copy of already existing code and this PR is not gonna be merged. As for the memory usage I measured it and it looks fine (below 70MB for everything in this script with a set of 400k orders and 450k order lines)